src/platform/published/pal.h

```diff
@@ -192,6 +184,58 @@ extern "C"
     };
 #endif
 
+#ifdef __cplusplus
+    enum class xlang_result : uint32_t
+    {
+        xlang_success = 0,
```

Comment:
>xlang_success [](start = 8, length = 13)

we should be consistent on case convention.  also note that c++ enum classes enforce scope, so the "xlang_" prefix is redundant.  (see xlang_string_encoding for both)

Reply:
I agree, I will address the removal of the xlang prefix.  I am going to leave the casing for now as that is something we need to come to an agreement on for interfaces and this (the pal functions use snake casing while interfaces use pascal casing).  I opened a issue to track that.

Reply:
Addressed.  Tracking casing as part of separate bug.